### PR TITLE
More docs fixes

### PR DESCRIPTION
**Pull request by jechter**
-Retake all screenshots with latest Input System and 19.3 UI
-New Screenshots for UI components
-Removed Future page (I don't think docs is the right place for this, also the future is too uncertain atm to commit to this in the docs)
-Feedback from last docs PR
-Fix compiling tests in 19.3
-Some 19.3 UI tweaks
-Some tweaks to existing docs pages

### CODE REVIEW

File: Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md [hunk 1]
```diff
@@ -29,61 +29,33 @@ The following devices are supported in a way that does not require recognition o
...
-
-|Chrome(Win)|Chrome(Mac)|Chrome(Linux)|Firefox(Win)|Firefox(Mac)|....|
-|-----------|-----------|-------------|------------|------------|----|
-
-### Steam
-
-    NOTE: Steam controller API support is still work-in-progress.
-
-For details about Steam controller API support, please consult the [Steam Support Guide](Steam.md).
+The input system supports the "Standard Gamepad" mapping as specified in the [W3C Gamepad Specification](https://www.w3.org/TR/gamepad/#remapping). We also support gamepads and joysticks that are surfaced by the browser without a mapping, but that support is generally limited to detecting axes and buttons which are present, without any context as to what they mean, which means that this is generally only useful when [manually remapped by the user](HowDoI.md#-create-a-ui-to-rebind-input-in-my-game).
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md]: For the latter part, might be worth clarifying how we surface them. I assume gamepads come out as Joysticks in that case? (like with HID)

File: Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md [hunk 2]
```diff
@@ -29,61 +29,33 @@ The following devices are supported in a way that does not require recognition o
...
-## XR
+## Other Gamepads, Joysticks and Racing Wheels
 
-|Device|Windows|Mac|Linux|UWP|Android|iOS|Xbox|PS4|Switch|WebGL|
-|------|-------|---|-----|---|-------|---|----|---|------|-----|
-|Vive|
-|Rift|
-|Daydream|
-|WMR|
+The input system supports any devices which implement the USB HID specification. However, for devices which don't have specific [layouts](Layouts.md) implemented in the input system, we can only surface the information available from the HID descriptor of the device, which limits how precisely we can describe a control. These devices often work best when allowing the user to manually [manually remap the controls](HowDoI.md#-create-a-ui-to-rebind-input-in-my-game). If you need to support a specific device, you can also [add your own mapping for it](HowDoI.md#-create-my-own-custom-devices). See the [manual page on HID](HID.md) for more information.
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md]: "which implements"

File: Packages/com.unity.inputsystem/Documentation~/Testing.md [hunk 3]
```diff
@@ -4,11 +4,8 @@ The input system has built-in support for writing automated tests involving inpu
 
 ## Setting Up Test Assemblies
 
->NOTE: At the moment, the tests from the input system itself will be injected into user projects when following the steps here. We're working on fixing that.
+Using test support requires setting up a test assembly for your tests. To do so, create a new assembly definition ("Create >> Assembly Definition") and add references to `nunit.framework.dll`, `UnityEngine.TestRunner`, and `UnityEditor.TestRunner` (as described [here](https://docs.unity3d.com/Packages/com.unity.test-framework@1.0/manual/workflow-create-test-assembly.html)) as well as `UnityEngine.Input.dll` and `UnityEngine.Input.TestFramework.dll` for the input system.
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Documentation~/Testing.md:7]: Are the TestRunner references required now? Odd setup I'm thinking...

**Review by Rene-Damm — APPROVED** [review]

File: Packages/com.unity.inputsystem/Documentation~/Testing.md (hunk 3 above)
Comment (jechter) [reply Packages/com.unity.inputsystem/Documentation~/Testing.md:7]: Yes.

**Pull request merged**